utils/wasm-instrument/src/gas_metering/schedule.rs

```diff
@@ -757,3 +757,23 @@ pub struct Weight {
     pub ref_time: u64,
     pub proof_size: u64,
 }
+
+pub struct ExtraWeights {
```

Comment:
You need to modify `pallets/gear/src/schedule.rs` and add new weights there, and the `weight-diff` utility will dump the weights to `utils/wasm-instrument/src/gas_metering/schedule.rs`.